test/test_tensor_creation_ops.py

```diff
@@ -2373,6 +2387,20 @@ def test_logspace_vs_numpy(self, device, dtype):
             self.assertEqual(t[0], a[0])
             self.assertEqual(t[steps - 1], a[steps - 1])
 
+    def test_logspace_vs_numpy_complex(self, device):
+        dtype = torch.complex64
```

Comment:
Analogous comments here as in previous test - seems like the tests could be combined?